Dockerfile

```diff
@@ -32,10 +33,11 @@ RUN curl -LO ${SBCL_URL} && \
     tar -xf sbcl-${SBCL_VERSION}-source.tar.bz2 && \
     rm /src/sbcl-${SBCL_VERSION}-source.tar.bz2 && \
     cd /src/sbcl-${SBCL_VERSION} && \
-    bash make.sh --fancy && \
+    ln -s /src/sbcl-${SBCL_VERSION} /src/sbcl && \
```

Comment:
is this softlink required for the build? just curious

Reply:
It makes life easier. The quilc build isn't necessarily aware of the sbcl version, and so having a canonical path is simpler.